students/grzegorz_bajorski/lesson_02_flow_control/ladder.py

```diff
@@ -0,0 +1,7 @@
+n = int(input())
+
+for j in range(1,n+1):
+    for i in range(1,j+1):
+        print(i, end='')
+    print(sep='\n')
+
```

Comment:
Too many empty lines at the end of the file. According to PEP8 should be only one.